src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/TemplateTargetExtension.cs

```diff
@@ -9,9 +9,15 @@ namespace Microsoft.AspNetCore.Razor.Language.CodeGeneration
     internal class TemplateTargetExtension : ITemplateTargetExtension
     {
         public static readonly string DefaultTemplateTypeName = "Microsoft.AspNetCore.Mvc.Razor.HelperResult";
+        public static readonly string DefaultPushWriterMethod = "PushWriter";
+        public static readonly string DefaultPopWriterMethod = "PopWriter";
 
         public string TemplateTypeName { get; set; } = DefaultTemplateTypeName;
 
+        public string PushWriterMethod { get; set; } = DefaultPushWriterMethod;
```

Comment:
Why do we need the properties? SHouldn't we just call the methods on basicwriter? or are they not exposed?